packages/gatsby-theme-carbon/src/styles/internal/_page.scss

```diff
@@ -134,6 +134,15 @@ img {
   padding-right: 0;
```

Comment:
Or, on second thought (to my above comment https://github.com/carbon-design-system/gatsby-theme-carbon/pull/864/files#r434771615):

It this means specifically "there will be no gutter on small breakpoints", then I think this should be wrapped in a media query like so:

```css
.container .#{$prefix}--no-gutter-sm {
  @include carbon--breakpoint-down("sm") {
    padding-left: 0;
    padding-right: 0;
  }
}
```

Because then it means it's only being applied on sm breakpoints like the prop implies. Which means, that if the prop is excluded, it should have a gutter, right?

Then for aside minis, they would only need to add `noGutterSm` bc by default lg breakpoint will have a gutter...

Reply:
I'm assuming that the default (w/o these noGutter styles) means that the column has a gutter.

Reply:
@vpicone any thoughts on this?

Reply:
Typically the breakpoint stuff works is 'starting with this and all the way up' so you don't have to define it again and again through all the of the breakpoints.

The change above would add gutters to any breakpoints larger than small which might be an unexpected change.

Reply:
Ah, I thought the prop `noGutterSm` meant no gutters only at that breakpoint. I see now, it means **starting** at that breakpoint.